python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -531,6 +540,10 @@ def run(self, inputExposure, refObjLoader=None, dataId=None, skyCorr=None):
             badMaskPlanes=self.config.badMaskPlanes,
             discardNanFluxObjects=(self.config.discardNanFluxStars),
         )
+        # Dont create empty fits files if there is no normalized stamp!
+        if not len(brightStarStamps._stamps) > 0:
```

Comment:
Is there a good reason why this cannot be simply:
```
if len(brightStarStamps._stamps) == 0:
```

Reply:
Good point!
Modification applied.

Reply:
len is guaranteed to return an integer >=0 in Python (it raises a ValueError otherwise), so checking equals zero is fine. In other cases you may need to be more careful if comparing to a number that might be nan.

Reply:
Or:

```python
if not brightStarStamps._stamps:
    ...
```
since a list will evaluate to False if it's empty.

Reply:
Good point! I implemented @timj suggestion as it looks and efficient and safe way for that.